src/Model/AutoLoginKey.php

```diff
@@ -91,29 +93,24 @@ public static function get_by_key( $key ) {
 			return null;
 		}
 
-		if ( (bool) $row['one_time'] ) {
-			self::maybe_delete_one_time_key( $key );
-		}
-
 		return new self( $row );
 	}
 
 	/**
-	 * Static method to check if a key is one-time and to delete it from the
-	 * database if it is.
+	 * Check if the key is one-time and to delete it from the database if it is.
 	 *
-	 * @param string $key
+	 * @param AutoLoginKey $key
```

Comment:
Method no longer takes this param

Reply:
Good spot. Thanks. Why didn't my editor pick this up? Hmm...